### PR TITLE
fix over array of objects not finding class

### PR DESCRIPTION
**Pull request by schojnicki-sembot**

### CODE REVIEW

**Comment by cedricziel** [comment]
Thank you, @schojnicki-sembot !

Can you explain what this change does?

**Comment by slnw** [comment]
should be `str_replace` in case package isn't being used in Laravel.

File: src/Client/HttpClientTrait.php [hunk 1]
```diff
@@ -57,6 +58,7 @@ protected function mapResponse(ResponseInterface $response, array $map): object|
         if (str_contains($map[$responseCode], '[]')) {
             $objects = [];
             foreach ($body as $item) {
+                Str::replace('[]', '', $map[$responseCode]);
```
Comment (slnw) [inline src/Client/HttpClientTrait.php]: $map[$responseCode] = str_replace('[]', '', $map[$responseCode]);

**Review by slnw — CHANGES_REQUESTED** [review]

**Comment by slnw** [comment]
> problem class[] cannot be found therefore i wanted to strip the [] from class name so it will create an array of objects in given class

should be `$map[$responseCode] = str_replace('[]', '', $map[$responseCode]);`

**Comment by slnw** [comment]
@cedricziel pls merge

**Pull request merged**